refinerycms-search.gemspec

```diff
@@ -8,6 +8,7 @@ Gem::Specification.new do |s|
   s.homepage          = %q{http://refinerycms.com}
   s.email             = %q{info@refinerycms.com}
   s.authors           = ["Philip Arndt", "Uģis Ozols", "Joe Sak"]
+  s.metadata          = { "contributor" => "Brice Sanchez <https://github.com/bricesanchez>" }
```

Comment:
So please remove this line. I don't want to keep contributors list in gemspec.

We have list of contributors available at https://github.com/refinery/refinerycms-search/graphs/contributors.